packages/flutter_tools/test/commands.shard/hermetic/run_test.dart

```diff
@@ -1241,7 +1270,7 @@ void main() {
         throwsToolExit(),
       );
 
-      final DebuggingOptions options = await command.createDebuggingOptions(true);
+      final DebuggingOptions options = await command.createDebuggingOptions();
```

Comment:
On `drive_test.dart`, `command.createDebuggingOptions(false)` was replaced by no parameters (OK), but here, the call is `createDebuggingOptions(true)`. Why?